package.json

```diff
@@ -6,16 +6,15 @@
   "scripts": {
     "build": "npm run build:css",
     "build:css": "node-sass --output-style=expanded --output=server/public/build/stylesheets client/sass",
-    "pretest": "npm run test:lint",
-    "start:watch": "nodemon --inspect=0.0.0.0 --ext js,njk --legacy-watch index.js",
-    "test": "npm run test:unit",
+    "test": "npm run test:lint && npm run test:unit",
     "test:lint": "standard",
-    "test:unit": "lab -c -r console -o stdout -r junit -o ./test-output/junit.xml"
+    "test:unit": "jest --forceExit",
+    "unit:watch": "jest --coverage=false --onlyChanged --watch --forceExit",
```

Comment:
are the forceExits required in the tests? The test script seem to run OK without it.

Reply:
Will remove